redbeams/src/main/java/com/sequenceiq/redbeams/RedbeamsApplication.java

```diff
@@ -3,26 +3,13 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.actuate.autoconfigure.metrics.web.servlet.WebMvcMetricsAutoConfiguration;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 
 import springfox.documentation.swagger2.annotations.EnableSwagger2;
 
 @EnableSwagger2
-@SpringBootApplication(scanBasePackages = {"com.sequenceiq.redbeams",
-        "com.sequenceiq.cloudbreak.auth",
-        "com.sequenceiq.cloudbreak.auth.altus",
-        "com.sequenceiq.cloudbreak.auth.filter",
-        "com.sequenceiq.cloudbreak.auth.security",
-        "com.sequenceiq.cloudbreak.security",
-        "com.sequenceiq.cloudbreak.api.util",
-        "com.sequenceiq.cloudbreak.conf",
-        "com.sequenceiq.cloudbreak.config",
-        "com.sequenceiq.cloudbreak.cache.common",
-        "com.sequenceiq.cloudbreak.service.secret",
-        "com.sequenceiq.cloudbreak.common.database",
-        "com.sequenceiq.cloudbreak.common.service",
-        "com.sequenceiq.cloudbreak.common.dbmigration",
-        "com.sequenceiq.cloudbreak.validation",
-        "com.sequenceiq.cloudbreak.common.converter"},
+@EnableJpaRepositories(basePackages = { "com.sequenceiq.redbeams", "com.sequenceiq.flow" })
+@SpringBootApplication(scanBasePackages = {"com.sequenceiq"},
```

Comment:
This has the potential to scan a ton of stuff that we don't want to include, even from random other services. I highly prefer keeping this narrowed down to specific packages that we know we need.

Reply:
This is kind of outside of the pattern that every other microservice uses, however. Additionally this gets really hairy with flow, as it pulls in a lot of unexpected dependencies.

Reply:
I highly agree, this should be more specific.

Reply:
I have been overrulled! Will change.